test/data/testSP.py

```diff
@@ -359,10 +359,15 @@ def testmetadata_package_version(self):
     @skipForParser('Psi4', 'reading cpu/wall time is not implemented for this parser') 
     @skipForParser('Turbomole', 'reading cpu/wall time is not implemented for this parser') 
     def testmetadata_times(self):
-        """Does metadata have expected keys and values?"""
-        self.assertIn("cpu_time", self.data.metadata)
-        self.assertIn("wall_time", self.data.metadata)
-
+        """Does metadata have expected keys and values of correct types?"""
+        if "wall_time" in self.data.metadata:
+            assert self.data.metadata["wall_time"]

```

Comment:
This test doesn't seem like it will ensure that `wall_time` is in `metadata`. Shouldn't this (or a similar line to what was removed) appear outside of the if statement?

Reply:
Same for `cpu_time`

Reply:
This was changed to cover files where either or both of the properties can't be parsed. In which case `wall_time` or `cpu_time` does not appear in metadata at all, rather than being present but as an empty list for example.

Reply:
Thanks that makes sense